sdk/labservices/Microsoft.Azure.Management.LabServices/src/Properties/AssemblyInfo.cs

```diff
@@ -6,8 +6,8 @@
 
 [assembly: AssemblyTitle("Microsoft Azure Lab Services Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Lab Services Resources.")]
-[assembly: AssemblyVersion("4.0.0.0")]
-[assembly: AssemblyFileVersion("4.0.0.0")]
+[assembly: AssemblyVersion("4.1.0.0")]
```

Comment:
Not only this file, please follow https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process?anchor=track-1-sdk-generation-from-updated-spec to update all

Reply:
Hi @ArthurMa1978, Is it the Assembly version that needs an update here?

Reply:
updated the Assembly version to beta version.